packages/OptimizeImage/OptimizeImage.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import ResponsiveImage from '@tds/core-responsive-image'
+import { safeRest } from '@tds/util-helpers'
+import checkWebpFeature from './checkWebpFeature'
+
+/**
+ * @version ./package.json
+ */
+const OptimizeImage = ({ contentfulAssetUrl, alt, quality, xs, sm, md, lg, xl, ...rest }) => {
+  // states used to ensure images are downloaded only once
+  const [hasLoaded, setHasLoaded] = useState(false)
+  const [imgUrls, setImgUrls] = useState({})
+
+  useEffect(() => {
+    if (!contentfulAssetUrl.match(/.svg/g)) {
+      // Currently not all browsers support webP
+      checkWebpFeature(result => {
+        let format = ''
+        if (result) {
+          format = 'fm=webp'
+        } else if (contentfulAssetUrl.match(/.jpeg/g)) {
```

Comment:
If its not a webp or a jpeg, what will it default to?  How will this handle pngs (pngs need to be supported)

Reply:
png's don't need further formatting and can be used as is.